lib/cmd/common/import.js

```diff
@@ -122,12 +123,19 @@ function importAppNgui(projectId, appTitle, appTemplateType, fileOrRepo, deployE
       connections:[],
       template: {
         repoUrl: repo,
-        type: appTemplateType
+        type: appTemplateType,
+        imported: true,
+        repoBranch: 'refs/heads/' + repoBranch
       }
     };
 
     var url = 'box/api/projects/' + projectId + '/apps';
-    common.doApiCall(fhreq.getFeedHenryUrl(), url, payload,  i18n._("Error importing git repo: "), cb);
+    common.doApiCall(fhreq.getFeedHenryUrl(), url, payload, i18n._("Error importing git repo: "), function(err, data) {
+      if (err) return cb(err);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)